src/components/interactive/dragBoxLayer.ts

```diff
@@ -155,14 +146,15 @@ export module Component {
         var r = bounds.bottomRight.x;
         var rad = this._detectionRadius;
 
+        var resizable = this.resizable();
```

Comment:
It seems like
This seems like
```
if (!this.resizable()) {
  return edges;
}
```

would work here and would skip all the complexity below.  Is this true?

English-wise, this would literally read, if the box isn't resizable then there are no resizing edges gotten